src/App.css

```diff
@@ -13,6 +13,20 @@
   }
 }
 
```

Comment:
Apps is working as expected, and the design is coming up pretty well 👌 There are however let's endeavor to improve our design by making the following adjustments:
- [ ] The category input field is missing, let's ensure to add this, as seen in the template design.
- [ ] The user at the top right of the navbar is overflowing the page, let's ensure that this is contained within the page and properly padded, as seen in the template design.
- [ ] Let's remove the black border around the _Update Progress_ button, the button label should be centered within the button, like the one seen in the template design.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/227608898-d1b073d3-841e-47e5-b0a8-eefe2b6fd5a0.png)

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/227609451-37f289a1-6ecd-4de1-837e-9305b1cca32c.png)

**The original design:**
![image](https://user-images.githubusercontent.com/12745474/227609686-4f38bfe7-5ae4-4234-b17b-77b094285e86.png)

Reply:
Hi @rloterh,

Thank you for your feedback on my code's design. I've implemented the changes you suggested and added the missing category input field, properly padded the user at the top right of the navbar, and removed the black border around the Update Progress button while centering its label. I appreciate your valuable input and will keep you updated on any further improvements.